app/src/main/java/io/novafoundation/nova/app/root/presentation/splitScreen/SplitScreenFragment.kt

```diff
@@ -43,14 +49,37 @@ class SplitScreenFragment : BaseFragment<SplitScreenViewModel>() {
     }
 
     override fun initViews() {
+        dappEntryPoint.setOnClickListener { viewModel.onTabsClicked() }
+        dappEntryPointClose.setOnClickListener { viewModel.onTabsCloseClicked() }
     }
 
     override fun subscribe(viewModel: SplitScreenViewModel) {
+        setupCloseAllDappTabsDialogue(viewModel.closeAllTabsConfirmation)
+
+        viewModel.dappTabsQuantity.observe {
+            val shouldBeVisible = it > 0
+            val isVisibilityChanged = dappEntryPoint.isVisible != shouldBeVisible
+
+            if (isVisibilityChanged) {
+                mainNavHost.outlineProvider = RoundCornersOutlineProvider(getOutlineCornerRadius(shouldBeVisible))
+            }
+
+            dappEntryPoint.isVisible = it > 0
+            dappEntryPointText.text = getString(R.string.dapp_entry_point_title, it)
```

Comment:
Shouldnt we display a proper tab name in case there's only one tab?
AlsoI think this formatting logic can go to ViewModel (whether to make entryPointVisible, what text to set to entry point)